### PR TITLE
Update and regenerate all APIs to use self-signed JWTs

### PR DESCRIPTION
**Pull request by jskeet**
Currently this leaves Compute using self-signed JWTs. That appears to work, but we need more confirmation before the next release.

### CODE REVIEW

File: tools/Google.Cloud.Tools.ReleaseManager/UpdateDependencies.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
+﻿// Copyright 2020 Google LLC
```
Comment (amanda-tarafa) [inline tools/Google.Cloud.Tools.ReleaseManager/UpdateDependencies.cs:1]: tiny nit: 2021 (not sure why I can add that as a suggestion)

File: apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableServiceApiClientPartial.cs [hunk 2]
```diff
@@ -234,17 +227,18 @@ protected override async Task<CallInvoker> CreateCallInvokerAsync(CancellationTo
             }
         }
 
-        /// <inheritdoc />
-        protected override bool CanUseChannelPool => false;
+        // Note: GetChannelPool() will still
```
Comment (amanda-tarafa) [inline apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableServiceApiClientPartial.cs:230]: nit: incomplete note

**Review by amanda-tarafa — APPROVED** [review]
LGTM

**Comment by jskeet** [comment]
Okay, with the fix in #6967 the diff should pass...

**Pull request merged**